### ISSUE TITLE
How to set config to File Transfer Progress Status

### ISSUE THREAD

**Issue by TongTC** [virtualmin/authentic-theme#1576]
Dear @iliarostovtsev 

Good Afternoon from Thailand.

Please refer to the picture below

![Screenshot 2022-01-13 130500](https://user-images.githubusercontent.com/56378146/149274845-1d146b5a-a0bf-4dba-b1f4-46e9621d901b.png)

When I upload the files to server by not root permission.

The File Transfer Progress status not show.

Could you suggest me how to set config for File Transfer Progress status to show ?

Thank you so muchhh

Best Regard.

**Comment by TongTC** [comment]
Dear All

OK, I can fix it this issue.

I have editted permission in theme directory.

Best Regard

**Comment by iliarostovtsev** [comment]
> Good Afternoon from Thailand.

Good afternoon. 🙂 

> Could you suggest me how to set config for File Transfer Progress status to show ?

File progress automatically shows for a large file uploads, when upload takes time.

---

I can see that you have latest devel version of the theme version installed -- beware it will may not work correctly with uploads as it requires [latest relevant patches](https://github.com/webmin/webmin/commit/60a5ea924e95cc0434d29dfc7131069f4a30dd9b) for Webmin repo as well.

**Comment by TongTC** [comment]
Dear @iliarostovtsev 

Yes, I upload the large file around   300 MB.

Sometime Webmin finished upload but  file that upload  is  uploaded around 100MB( actually not success ).

Please refer the picture for Authentic Theme version 

<img width="758" alt="Screenshot 2022-01-13 184707" src="https://user-images.githubusercontent.com/56378146/149324827-c860c7f9-1385-4bb7-8c78-01e5658ec254.png">

Best Regard

**Comment by iliarostovtsev** [comment]
I am confused as based on your very first screenshot yellow delete button (move to trash mode) was only added few days ago in a development version.

I do see the progress when logged in as a `root` user and as virtual server owner:

![image](https://user-images.githubusercontent.com/4426533/149332646-63b93ed8-97ba-4f86-903c-033f14f5050c.png)

> When I upload the files to server by not root permission.

What do you mean by this exactly? How can I reproduce a problem?

**Comment by TongTC** [comment]
Dear @iliarostovtsev 

Sorry, It may  confuse you.

> What do you mean by this exactly? How can I reproduce a problem?

- This is another problem.

**Refer your picture that you mention it is another computer and I am checking the CSS for fix it.**

For  this topic is another one computer (Authentic Theme version 19.84.7)  I  used  not root permission user   for large uploading file  but  transfer progress status not show 


Thank you for your support.
Best Regard

**Comment by iliarostovtsev** [comment]
@jcameron Jamie, any clue why **`uptracker.cgi`** returns **`document.forms[0].pc.value = "Not started";`** as upload started when logged is as Safe Webmin user?

**Comment by jcameron** [comment]
Because a safe webmin user isn't allowed to upload files, so the upload tracker never has anything to track the progress of?

**Comment by TongTC** [comment]
Hello @jcameron 

> Because a safe webmin user isn't allowed to upload files, so the upload tracker never has anything to track the progress of?

- yes , when I upload the large file upload tracker do not show any progress.

Thank you for your response.

Best Regard

**Comment by iliarostovtsev** [comment]
> Because a safe webmin user isn't allowed to upload files, so the upload tracker never has anything to track the progress of?

Uploads are allowed with the safe Webmin user.

Perhaps, the problem for regular user is to write a temp file to `$ENV{'WEBMIN_VAR'}` dir:

```
# Find the location of the user's upload progess file
...
$upfile = "$ENV{'WEBMIN_VAR'}/upload.$id";
```

We probably need to pass a `uid` when uploading as a user.

**Comment by iliarostovtsev** [comment]
Thanks for reporting. It will be fixed in the next Authentic Theme release 19.85.

**Comment by TongTC** [comment]
Thank you all of you for  contribution
